test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerMockTest.java

```diff
@@ -822,6 +802,48 @@ public void signInV4ThrowsUnauthoried() throws Exception {
         controller.signInV4();
     }
     
+    @Test
+    public void unconsentedSignInSetsCookie() throws Exception {
+        Http.Response mockResponse = mockPlayContextWithJson(EMAIL_PASSWORD_SIGN_IN_REQUEST);
+        when(authenticationService.signIn(any(), any(), any())).thenThrow(new ConsentRequiredException(userSession));
+        
+        try {
+            controller.signInV4();
+            fail("Should have thrown exeption");
+        } catch(ConsentRequiredException e) {
+            verify(mockResponse).setCookie(BridgeConstants.SESSION_TOKEN_HEADER, TEST_SESSION_TOKEN,
+                    BridgeConstants.BRIDGE_SESSION_EXPIRE_IN_SECONDS, "/");
```

Comment:
Can we leverage verifyCommonLoggingForSignIns() in this test (and in the next 3 tests)? This keeps our tests consistent and complete.

Reply:
Yes, switched the tests that are verifying we log when a ConsentRequiredException is thrown, so they all use verifyCommonLoggingForSignIns().